index.js

```diff
@@ -33,10 +35,12 @@ function openConfig (cb) {
     return tryOpenConfig(path.join(os.tmpdir(), configfile), cb);
   }
 
-  tryOpenConfig(path.join(userHome, configfile), function (err, fd) {
-    if (err) return tryOpenConfig(path.join(os.tmpdir(), configfile), cb);
-    return cb(null, fd);
-  });
+  fs.mkdir(configPath, function () {
```

Comment:
Yes I understand @phated meant nested directories. With my reply I meant that it's nested inside a default OS directory so I'm wondering if the case needs to be handled.

Regarding error handling, if for some reason we can't create the directory the next function `tryOpenConfig` will handle the logic of falling back to a temporary directory so that's why there is no need to handle it.